docs/scarpet/api/Auxiliary.md

```diff
@@ -14,7 +14,6 @@ and `voice`. `pos` can be either a block, triple of coords, or a list of three n
  
 Used with no arguments, return the list of available sound names.
  
-Throws `unknown_sound` if sound doesn't exist.
```

Comment:
Instead this should (in some way) mention that it won't throw if a sound isn't a vanilla one (or isn't present on the server or whatever used to trigger this), and also mention that the list of available sounds above won't include those.

Reply:
done